assembly_and_binning/tutorial_assembly_and_binning.md

```diff
@@ -0,0 +1,635 @@
+Which is actually a sort of high bar for stringing 17 tools together!
+
+## Instance hygiene
+
+Metagenomics workflows can take up quite a bit of space. 
+We want to make sure that we have enough space and compute power on our instances to execute the whole thing. 
+
+We need three things to run a workflow: hard drive space, RAM, and CPUs. 
++ **hard drive space** is physical, persistent storage. 
+We write our files to disk for long term storage
++ **RAM** is working memory. 
+It temporarily stores all the information that your computer needs to work right that minute. 
+Some tools need to load all of your data into RAM in order to work with it.
+Assembly is one of these steps -- to assemble something, you need at minimum the amount of RAM as you have in gigabytes of data.
++ **CPUs** are the electronic circuitry within a computer that carries out the instructions of a computer program by performing the basic arithmetic, logic, controlling, and input/output (I/O) operations specified by the instructsion (see wikipedia [article here](https://en.wikipedia.org/wiki/Central_processing_unit)).
+
+Log into your jetstream instance using `ssh` or the jupyter lab url.
+
+If you don't have access to a jetstream instance, you can use this Pangeo binder link instead: [![Binder](https://aws-uswest2-binder.pangeo.io/badge_logo.svg)](https://aws-uswest2-binder.pangeo.io/v2/gh/taylorreiter/stamps2022-binder/main)  
+
+When you have a remote computer up and running, open a terminal.
+Let's check the amount of hard drive space we have. 
+From your bash prompt, run:
+
+```
+df -h .
+```
+
+You should see an output like this:
+
+```
+Filesystem      Size  Used Avail Use% Mounted on
+/dev/sda1        58G   15G   44G  25% /
+```
+
+You need at least 10G of space on your instance. 
+If you don't have that much, put up a pink sticky and a helper will come around to help remove files. 
+
+Let's also take a look at how much ram we have available:
+
+```
+free -mh
+```
+
+and you'll see something like this:
+```
+               total        used        free      shared  buff/cache   available
+Mem:            29Gi       894Mi        27Gi       1.0Mi       1.2Gi        28Gi
+Swap:             0B          0B          0B
+```
+
+And the amount of CPUs:
+
+```
+nproc
+```
+
+Which should return:
+
+```
+8
+```
+
+Ok! We now have an idea of how much compute is available to us. 
+In general, it's really hard to know how much of each resource you need for each tool before you run it. 
+You may be able to google around for this information, but you will also develop this intuition over time as you run more tools. 
+We have enough compute to execute our tutorial because we are working with a subsampled dataset. 
+
+The last bit of instance hygiene we need to take care of is making sure that we all have the right channels, or package managers, enabled for conda. 
```

Comment:
Are you intro'ing conda/mamba before this?

Reply:
im just going to do it while teaching.

Reply:
i dont feel a strong pull to have all of the things i want covered actually written down